docs/configure

```diff
@@ -407,11 +400,6 @@ acl_check_rcpt:
 
   accept  hosts         = +relay_from_hosts
 
```

Comment:
Oh, I see what the "Again" is for. Perhaps the block above should also be moved upwards?